orca/orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/api/BakeryServiceSpec.groovy

```diff
@@ -60,14 +75,23 @@ class BakeryServiceSpec extends Specification {
 
   def mapper = OrcaObjectMapper.newInstance()
 
-  def setupSpec() {
+  def setup() {
+    actualUrl = new BlockingVariable<String>(5)
+    actualPayload = new BlockingVariable<String>(5)
```

Comment:
Can we switch back to setupSpec (to speed up the tests) and then use wiremock verify to assert on the request body that wiremock receives?  I think that'll also be simpler.  See e.g. [here](https://github.com/spinnaker/spinnaker/blob/66b9ab017406ef1565b20c5c04acf18bbcb4cdcd/gate/gate-web/src/test/java/com/netflix/spinnaker/gate/service/EchoServiceTest.java#L110-L112).

Reply:
Will do...